localstack/utils/docker_utils.py

```diff
@@ -80,16 +81,17 @@ def __init__(self, network_name: str, message=None, stdout=None, stderr=None) ->
         self.network_name = network_name
 
 
+class HashableList(list):
```

Comment:
Coincidentally also made a similar change in https://github.com/localstack/localstack/pull/4971/files#diff-24795d42564e9e1ca647c5bed173eb3ce0f32fd8b2c2d184c20112ac354587e1R466 ;) , as we're re-using this class elsewhere.. (we may need to rebase your branch onto latest `master`)

Reply:
fixed